### ISSUE TITLE
admin is not in the sudoers file by default

### ISSUE THREAD

**Issue by xeruf** [YunoHost/issues#2091]
### Describe the bug

https://yunohost.org/en/ssh
> If you connected as admin and would like to become root for convenience (e.g. to avoid typing sudo in front of every command), you can become root using the command sudo su or sudo -i.

But I can't use sudo as admin by default.

```sh
admin@host:~$ sudo su
[sudo] password for admin: 
admin is not in the sudoers file.  This incident will be reported.
```

### Context

- Hardware: VM on own server
- YunoHost version: 11.0.10.2
- I have access to my server: Everything
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance? no

**Comment by alexAubin** [comment]
Can you elaborate the context ... admin is not supposed to be in the sudoers file, because the sudo membership is managed by LDAP ... Any sane instance I've seen has admin able to run sudo ...

**Comment by xeruf** [comment]
The documentation says to run `sudo` as admin to gain root privileges. This should work out of the box.
It does not, as illustrated above.

**Comment by alexAubin** [comment]
As said it works for me for any Yunohost I have access to, so I don't know how to reproduce your issue, hence there seems to be something funky about your context ...

**Comment by xeruf** [comment]
Users > Groups and permissions
    
has no groups for me, is this supposed to be like that?

**Comment by alexAubin** [comment]
Nope ...

**Comment by zamentur** [comment]
Which install method did you use ?

**Comment by xeruf** [comment]
curl|bash on a fresh debian 11.3 LXC container created by Proxmox

**Comment by obrenckle** [comment]
Hi, I'm in a similar situation (admin can't sudo).
I installed yunohost 11.0.10.2 over a fresh Armbian 22.08.8 Bullseye with Linux 5.15.74-sunxi on a A20-OLinuXino lime2 with curl|bash.
`yunohost user list` only returns my user account (no admin account).
`yunohost user group list` returns all_users (with my account in it) and visitors (no members)

**Comment by GTHeyyy** [comment]
Hi!
The same here with a fresh install. I don't have this problem on my other old yunohost servers.

https://forum.yunohost.org/t/admin-is-not-in-the-sudoers-file-this-incident-will-be-reported/22441

**Comment by GTHeyyy** [comment]
Hi!
`ln -s /etc/ldap/ldap.conf /etc/sudo-ldap.conf` work for me

**Comment by obrenckle** [comment]
`ln -s /etc/ldap/ldap.conf /etc/sudo-ldap.conf` worked for me too

**Comment by alexAubin** [comment]
Fixed in https://github.com/YunoHost/yunohost/commit/d2417c33de53e8446bb25a003cb9f8bed6cbb7bb to be included in 11.1
